### ISSUE TITLE
Error When Setting N64 Settings

### ISSUE THREAD

**Issue by thecoreyburton** [TASVideos/BizHawk#888]
I was playing around with GLideN64 settings in the latest dev build and on occasion I get the following error message:

```
Unhandled exception has occurred in a component in your application. If you click Continue, the application will ignore this error and attempt to continue.

Object reference not set to an instance of an object.

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.NullReferenceException: Object reference not set to an instance of an object.
   at BizHawk.Client.EmuHawk.N64VideoPluginconfig.SaveSettings()
   at BizHawk.Client.EmuHawk.N64VideoPluginconfig.Button1_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

```

Usually this doesn't happen straight away (or when there's no config file). After starting a game, changing some settings and playing a movie though, upon trying to change any settings and apply them, I get that error message and cannot proceed without hitting "cancel". The only way to alter the settings at the current point is to delete the config file and start from scratch.

**Comment by thecoreyburton** [comment]
Does not seem to be present in 2.0.

**Comment by thecoreyburton** [comment]
In the process of trying to fix a few things for the Paper Mario publication, I ran into this again. Deleting the config file fixes it, but it gets tedious reconfiguring the emulator every time I want to try and make a slight change to the settings. This is present in the 2.1.0 release and can be seen by doing the following:

1 - Open a fresh copy of the emulator.
2 - Load the "Mario Story (J)" ROM.
3 - Start playback on this movie: http://tasvideos.org/5515S.html
4 - Stop playback on the movie after a few seconds, create a savestate.
5 - Quit the emulator.
6 - Re-open the emulator and ROM.
7 - Load the savestate.
8 - Open the N64 plugin settings and change a setting in GLideN64's options. In this case, I tried changing the aspect ratio from "Force 4:3" to "Adjust".
9 - Hit ok.

That should get you the error message to pop up. I know there's likely an easier way to reproduce the problem but these exact steps work fine.
